R/doAzureParallel.R

```diff
@@ -564,14 +564,16 @@ setHttpTraffic <- function(value = FALSE) {
       ),
       envir = .doAzureBatchGlobals,
       packages = obj$packages,
-      dependsOn = tasks,
+      dependsOn = list(taskIdRanges = list(list(start = 1, end = length(tasks)))),
       cloudCombine = cloudCombine,
       outputFiles = obj$options$azure$outputFiles,
       containerImage = data$containerImage
     )
     cat(". . .")
   }
 
+  rAzureBatch::updateJob(id)
```

Comment:
Why are you doing this? (add a comment). Also, I'm not sure I understand what is being updated? Do we need to pass in the job params?

Reply:
We need to update the job property for terminating the job after all the tasks are completed

Reply:
I think I get it, but the name of the function is quite confusing. Doesn't really say what is going on, and doesn't pass in any params.